### PR TITLE
[tracy] update to 0.8.2

### PR DESCRIPTION
**Pull request by alex-tdrn**
**Describe the pull request**

- #### What does your PR fix?
  Updates `tracy` to version 0.8.2. The vcxproj patch has been removed, since [it was merged upstream](https://github.com/wolfpld/tracy/pull/412/files#diff-a77fdc8f9d16eb55e3d0f19c1738839914856387facbeba1fc93bc81cf40f2a0)

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?
  all, No

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
  Yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?
  Yes

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

### CODE REVIEW

File: ports/tracy/portfile.cmake [hunk 1]
```diff
@@ -7,12 +7,11 @@ vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
...
     REPO wolfpld/tracy
-    REF 80f6a93da79af94fc4604e37439f5a75d5e7dfbd
-    SHA512 80d24fd969a9f8ac000b127c586745c85239139c50f6aacf625cea50551303031a7e14da01fd96a0e17d59ea639019cab119b76945e96cb6e7cfc7e5399252f8
+    REF a8511d357650282c6915f7aa9775e9c4945e42b5
+    SHA512 6e0ff0be7621490b607eaae46d4ec13ea66b0061ed580c99d75eda3a7c9479b34c70da9ad844bd4a05e329bb40cebf48e7c6bab05d2a946d56b2d432c7ddc125
     HEAD_REF master
     PATCHES
-        001-fix-vcxproj-vcpkg.patch
-        002-fix-capstone-5.patch
+        001-fix-capstone-5.patch
```
Comment (FrankXie05) [inline ports/tracy/portfile.cmake:14]: Note: This patch contains nothing.

**Review by FrankXie05 — CHANGES_REQUESTED** [review]

File: versions/baseline.json [hunk 2]
```diff
@@ -7901,4 +7901,4 @@
       "port-version": 1
     }
   }
-}
+}
```
Comment (Thomas1664) [inline versions/baseline.json:7904]: Please keep the final newline here...

File: versions/t-/tracy.json [hunk 3]
```diff
@@ -31,4 +36,4 @@
       "port-version": 0
     }
   ]
-}
+}
```
Comment (Thomas1664) [inline versions/t-/tracy.json:39]: ... and here as well.

File: ports/tracy/portfile.cmake [hunk 4]
```diff
@@ -7,11 +7,10 @@ vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO wolfpld/tracy
-    REF 80f6a93da79af94fc4604e37439f5a75d5e7dfbd
-    SHA512 80d24fd969a9f8ac000b127c586745c85239139c50f6aacf625cea50551303031a7e14da01fd96a0e17d59ea639019cab119b76945e96cb6e7cfc7e5399252f8
+    REF a8511d357650282c6915f7aa9775e9c4945e42b5
+    SHA512 6e0ff0be7621490b607eaae46d4ec13ea66b0061ed580c99d75eda3a7c9479b34c70da9ad844bd4a05e329bb40cebf48e7c6bab05d2a946d56b2d432c7ddc125
     HEAD_REF master
     PATCHES
-        001-fix-vcxproj-vcpkg.patch
```
Comment (FrankXie05) [inline ports/tracy/portfile.cmake:14]: Why remove this patch? Upstream did not make changes: 
https://github.com/wolfpld/tracy/blob/a8511d357650282c6915f7aa9775e9c4945e42b5/profiler/build/win32/Tracy.vcxproj#L100

**Review by FrankXie05 — CHANGES_REQUESTED** [review]

File: ports/tracy/portfile.cmake (hunk 4 above)
Comment (alex-tdrn) [reply ports/tracy/portfile.cmake:14]: Ah I'm very sorry, I messed up here. The vcxproj patch did not apply cleanly on 0.8.2, so I looked for changes upstream, saw that there was _some_ patch merged on that file related to vcpkg, so I assumed it was doing the same thing, but I see now that it doesn't, I should have been more careful. I'll update our patch when I have access to a windows machine to test things out with visual studio

Comment (alex-tdrn) [reply ports/tracy/portfile.cmake:14]: Alrighty, I've updated the patch, and tested the port with `.\vcpkg.exe install tracy[gui-tools]:x64-windows`
The resulting Tracy.exe starts and seems to work fine 👍 
Sidenote: looking at diffs of patches hurt my brain 🤮

**Review by FrankXie05 — APPROVED** [review]

**Comment by JavierMatosD** [comment]
Thanks!

**Pull request merged**